packages/yoroi-extension/features/connector-cardano.feature

```diff
@@ -10,7 +10,7 @@ Feature: dApp connector data signing
     Then I should see the connector popup
     And I select the only wallet named shelley-simple-15 with 5.5 balance
     Then The popup window should be closed
-    And The access request should succeed
+    And The access request should be succeed
```

Comment:
Just nitpicking: "succeed" is a verb so it should be "succeed" or "be successful".

Reply:
Got it. I will change it back.